content/spin/cli-reference.md

```diff
@@ -887,10 +1014,85 @@ SUBCOMMANDS:
 
 {{ blockEnd }}
 
+{{ startTab "v1.4"}}
+
+<!-- @selectiveCpy -->
+
+```console
+$ spin help
+
+The Spin CLI
+
+USAGE:
+    spin <SUBCOMMAND>
+
+OPTIONS:
+    -h, --help       Print help information
+    -V, --version    Print version information
+
+SUBCOMMANDS:
+    add          Scaffold a new component into an existing application
+    build        Build the Spin application
+    cloud*       Commands for publishing applications to the Fermyon Cloud.
+    deploy       Package and upload an application to the Fermyon Cloud.
+    doctor       Detect and fix problems with Spin applications
+    help         Print this message or the help of the given subcommand(s)
+    js2wasm*     A plugin to convert js files to Spin compatible modules
```

Comment:
I would not attempt to show plugins in this list (except for cloud, which is magically always shown if the plugin is not installed), as they will vary depending on what the user has installed.  Instead, add a paragraph saying "you may have see commands depending on what plugins you have installed, you can spot the wee devils by the asterisk"